sdk/communication/communication-callingserver/src/callingServerClient.ts

```diff
@@ -138,7 +144,7 @@ export class CallingServerClient {
     this.callingServerServiceClient = new CallingServerApiClient(url, pipeline);
     this.callConnectionRestClient = this.callingServerServiceClient.callConnections;
     this.serverCallRestClient = this.callingServerServiceClient.serverCalls;
-    this.downloadCallingServerApiClient = new CallingServerApiClientContext(url, pipeline);
+    this.deleteAndDownloadCallingServerApiClient = new CallingServerApiClientContext(url, pipeline);
```

Comment:
I was not a huge fan of `downloadCallingServerApiClient`, so I'm not a fan of appending `delete` to that name. Could we call it `apiClient` or something similar?

Reply:
Renamed to storageClient to make it a bit more clear.